src/main/scala/sbtecr/EcrPlugin.scala

```diff
@@ -25,7 +26,7 @@ object EcrPlugin extends AutoPlugin {
   override lazy val projectSettings = inConfig(ecr)(defaultSettings ++ tasks)
 
   lazy val defaultSettings: Seq[Def.Setting[_]] = Seq(
-    version := "latest",
```

Comment:
I think we can safely remove `import sbt.Keys.version` after removing this setting. Also, please update the `README.md` file as it still references `version` in the example.